resync/shared/shop_time_resolution.yaml

```
@@ -0,0 +1,21 @@
+# ShopTimeResolution view definition
+#   name: String!
+#   minutes_after_start: [Int64]
+#   time_resolution_minutes: [Int64]
+
+- name: default
+  minutes_after_start:
+    - 0
+    - 4320 # 3 days
+  time_resolution_minutes:
+    - 60
+    - 240 # 4 hours
```

```suggestion
- name: default
  minutes_after_start:
    - 0
  time_resolution_minutes:
    - 60
```
I want 60 minutes to be our default. It may make the runs in power-ops-staging take a lot longer, but if that becomes a problem, we can handle that by creating (and using) another ShopTimeResolution or just shortening the period.